resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -728,6 +728,7 @@ private[spark] class Client(
       new File(Utils.getLocalDir(sparkConf)))
     val confStream = new ZipOutputStream(new FileOutputStream(confArchive))
 
+    logDebug(s"Creating an archive with the config files for distribution at $confArchive.")
```

Comment:
Hi, @liupc . Thank you for your first contribution.
Please update the PR description. Currently, it still claims `IOException`.
```
This PR will catch the IOException and print some useful information.
```

Reply:
@dongjoon-hyun It's done! thanks!